convert/convert/config.py

```diff
@@ -29,22 +29,22 @@
 class Settings(BaseSettings):
     """Base settings values"""
 
-    minio_host: Optional[str] = os.getenv("MINIO_HOST")
-    minio_access_key: Optional[str] = os.getenv("MINIO_ACCESS_KEY")
-    minio_secret_key: Optional[str] = os.getenv("MINIO_SECRET_KEY")
+    minio_host: Optional[str] = os.getenv("S3_ENDPOINT_URL")
+    minio_access_key: Optional[str] = os.getenv("S3_ACCESS_KEY")
+    minio_secret_key: Optional[str] = os.getenv("S3_SECRET_KEY")
     s3_prefix: Optional[str] = os.getenv("S3_PREFIX")
-    s3_credentials_provider: Optional[str] = os.getenv(
-        "S3_CREDENTIALS_PROVIDER", "minio"
-    )
+    s3_credentials_provider: Optional[str] = os.getenv("S3_PROVIDER", "minio")
     uploading_limit: int = int(os.getenv("UPLOADING_LIMIT", 100))
     coco_image_format: str = "jpg"
     dpi: int = 300
     root_path: str = os.environ.get("ROOT_PATH", "")
-    assets_service_url: Optional[str] = os.getenv("ASSETS_SERVICE_URL")
-    job_service_url: Optional[str] = os.getenv("JOB_SERVICE_URL")
-    annotation_service_url: Optional[str] = os.getenv("ANNOTATION_SERVICE_URL")
-    taxonomy_service_url: Optional[str] = os.getenv("TAXONOMY_SERVICE_URL")
-    keycloak_url: Optional[str] = os.getenv("KEYCLOAK_URL")
+    assets_service_url: Optional[str] = os.getenv("ASSETS_SERVICE_HOST")
+    job_service_url: Optional[str] = os.getenv("JOBS_SERVICE_HOST")
+    annotation_service_url: Optional[str] = os.getenv(
+        "ANNOTATION_SERVICE_HOST"
+    )
+    taxonomy_service_url: Optional[str] = os.getenv("TAXONOMY_SERVICE_HOST")
+    keycloak_url: Optional[str] = os.getenv("KEYCLOAK_HOST")
```

Comment:
Could we rename those according to the new naming, `*_host`

Reply:
Renamed